### ISSUE TITLE
No //gloves option listed in witchyzon or javazon final build.

### ISSUE THREAD

**Issue by twiztiddreamz** [isid0re/SoloLeveling#367]
Just noticed that my witchyzon just did it's final respect.   It has obtained all the gear needed for the final build.  But when I checked the WitchyzonBuild.js  to double check all gear, it had no option for gloves in either WitchyzonBuild.js or JavazonBuild.js 

I tried equipping Laying of Hands, but it kept selling them off and it started using a crappy pair of gloves.   Can this be added or fixed so the bot doesn't keep doing this?

![WitchyzonBuild](https://user-images.githubusercontent.com/42836178/133450656-afa208e6-43f4-47ad-9360-0a6cbe669283.JPG)

**Comment by isid0re** [comment]
Yes you will need to add the respective pickit line to the finalbuild files autoequip tiers.
